sdks/python/apache_beam/yaml/pipeline.schema.yaml

```diff
@@ -85,7 +85,10 @@ $defs:
         items:
           properties:
             # Must be implicit.
-            input: { not: {} }
+            input:
+              oneOf:
+                - { "not": {} }
+                - { "enum": [ "__explicitly_empty__" ] }
```

Comment:
It feels like we shouldn't be validating once we've injected the `__explicitly_empty__` strings.